### ISSUE TITLE
Automatically check back in URLs after they have been checked out for two days

### ISSUE THREAD

**Issue by kmcculloch** [edgi-govdata-archiving/archivers.space#32]
_From @khdelphine on February 16, 2017 12:11_

Could we  automatically  check back in URLs after they have been checked out for two days?

And perhaps add a  comment in the note field like “This URL was automatically checked back in after two days so that it can be processed expeditiously.”

_Copied from original issue: b5/pipeline#69_

**Comment by kmcculloch** [comment]
Flagged as large because of the cron job nature of this, although I don't honestly know whether there's an easy way to define this kind of process in meteor. That said, I think it would be easier to just give admin users the ability to grab items back if they want to, and they could decide when to do it. Would that do for a start @khdelphine?

**Comment by khdelphine** [comment]
@kmcculloch: yes, allowing admins to check back in URLs for another user would work.  We could add in the documentation "Note that if you leave a URL checked out for more than 2 days, we might check it back in for you to enable another user to work on it." Or something of that nature.
